src/main/java/com/epam/dynamos3/LambdaOrdersHandler.java

```diff
@@ -1,201 +1,42 @@
 package com.epam.dynamos3;
 
-import com.amazonaws.services.dynamodbv2.document.Item;
-import com.amazonaws.services.dynamodbv2.document.ItemUtils;
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.LambdaLogger;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
 import com.amazonaws.services.lambda.runtime.events.StreamsEventResponse;
-import com.amazonaws.services.lambda.runtime.events.transformers.v1.DynamodbEventTransformer;
-import com.amazonaws.services.s3.AmazonS3;
-import com.amazonaws.services.s3.AmazonS3ClientBuilder;
-import com.amazonaws.services.s3.model.ObjectMetadata;
-import com.amazonaws.services.s3.model.S3Object;
-import com.amazonaws.services.s3.model.S3ObjectInputStream;
-import com.epam.dynamos3.pojos.Product;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import org.jsoup.Jsoup;
-import org.jsoup.nodes.Document;
-import org.jsoup.nodes.Element;
-import org.jsoup.select.Elements;
+import com.epam.dynamos3.services.FileService;
+import com.epam.dynamos3.services.FileServiceS3Impl;
 
 import java.io.*;
-import java.util.List;
-import java.util.stream.Collectors;
-
-import static j2html.TagCreator.*;
 
 public class LambdaOrdersHandler implements RequestHandler<DynamodbEvent, Serializable> {
 
-
-    private final String S3_BUCKET_NAME = "orders-lambda-html";
-    private final String PRODUCTS_FILE_NAME = "index";
-
     @Override
     public StreamsEventResponse handleRequest(DynamodbEvent input, Context context) {
 
         LambdaLogger LOGGER = context.getLogger();
 
-        List<Product> products = createProducts(input,context);
-
-        LOGGER.log("Connecting to S3 Bucket");
+        FileService fileService = new FileServiceS3Impl();
```

Comment:
Please, don't use *Impl naming, it is a bad practice and does not help to understand the real purpose of the class a better name could be: S3FileService

Reply:
Using *Impl is a flag to think about the Single Responsibility Principle, if the developer cannot give a descriptive name, probably that class does not have a single purpose.